larndsim/lightLUT.py

```diff
@@ -0,0 +1,119 @@
+"""
+Code adapted from https://github.com/PPKoller/ArgonCubeLUTSim/blob/main/module_0/lutSim.py
+"""
+
+import sys, time, math
+import argparse
+import numpy as np
+from math import log, isnan
+from . import consts    
+
+def GetLutGeometry(lut_path): # Finds the max and minimum values of the x,y and z axis and the number of voxels in each direction
```

Comment:
The description of the function goes below with triple quotes """, see https://www.python.org/dev/peps/pep-0257/. Also the name of the functions always start with a lowercase letter, so `getLutGeometry`. I prefer `this_convention` and not `thisConvention` for the name of the functions, but it's a matter of taste, we should just try to be consistent though. For the actual content of the docstring we adopted this format: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html, where you need to describe the argument and the return value of the function.

Reply:
@soleti can I suggest some information like this is put in the README under a "for developers" section?

Reply:
Sure, the convention in GitHub is to create a `CONTRIBUTING.md` file. I can do that when I have time (Mu2e CM this week...) or feel free to start one.

Reply:
Fixed in https://github.com/DUNE/larnd-sim/pull/39/commits/f06c570af59b5d185878076d571cfc8eb1b2b134.  We are happy to use `this_convention` even if `thisConventionIsBetter` ;)